tools/cli/src/main/java/org/apache/carbondata/tool/TablePrinter.java

```diff
@@ -17,26 +17,28 @@
 
 package org.apache.carbondata.tool;
 
-import java.io.PrintStream;
+import java.util.ArrayList;
 import java.util.LinkedList;
 import java.util.List;
 
 class TablePrinter {
```

Comment:
Now TablePrinter is not printing any details, it is adding to list. Please change the class name, Please check other places if applicable